subsystems/TypeHandlerLibrary/build.gradle.kts

```diff
@@ -6,7 +6,10 @@ plugins {
     `java-library`
 }
 
-apply(from = "$rootDir/config/gradle/common.gradle")
+apply(from = "$rootDir/config/gradle/publish.gradle")
+
+group = "org.terasology"
```

Comment:
Should subsystems be published to group "org.terasology.subsystems"?

Reply:
Examples from artifactory:

engine
```
<dependency>
    <groupId>org.terasology.engine</groupId>
    <artifactId>engine</artifactId>
    <version>4.1.0</version>
    <type>pom</type>
</dependency>
```

PC facade
```
<dependency>
    <groupId>org.terasology.facades</groupId>
    <artifactId>PC</artifactId>
    <version>4.1.0</version>
    <type>pom</type>
</dependency>
```

So maybe for type handler library subsystem we could opt for
```
<dependency>
    <groupId>org.terasology.subsystems</groupId>
    <artifactId>TypeHandlerLibrary</artifactId>
    <version>4.1.0</version>
    <type>pom</type>
</dependency>
```

Reply:
Concur, `org.terasology.subsystems` sounds great 👍